Doc/whatsnew/3.10.rst

```diff
@@ -596,6 +602,24 @@ Changes in the Python API
   a 16-bit unsigned integer.
   (Contributed by Erlend E. Aasland in :issue:`42393`.)
 
+* The ``loop`` parameter to nearly all :mod:`asyncio` functions has been removed
+  following deprecation in Python 3.8. For example, a coroutine that currently
+  look like this::
+
+     import asyncio
+
+     async def foo(loop):
+         await asyncio.sleep(loop=loop)
+
+  Can *usually* be replaced with this::
+
+     import asyncio
+
+     async def foo(loop):
+         await asyncio.sleep()
+
```

Comment:
I'd also add something along the lines of: "If it is specifically *not* intended to use the current running event loop (e.g. for using a loop in a separate OS thread), see :func:\`asyncio.run_coroutine_threadsafe()\`. However, simply removing the *loop* arg work as intended for the majority of use cases."

This should help to address the concern brought up by @vstinner in python-dev about helping users understand how to port their code. For most people, just removing the *loop* arg will have their code working as it was previously, potentially fix erroneous event loop usage, and improve readability.